test/collections/acme.goodies/tests/requirements.yml

```diff
@@ -0,0 +1,3 @@
+collections:
+  - name: ansible.posix
+    version: ">=1.0"
```

Comment:
As metioned in #ansible-devtools, **this is the wrong format**. `tests/requirements.yml` is **NOT** a Galaxy requirements file. Please look at all existing examples: https://github.com/search?q=org%3Aansible-collections+path%3A**%2Ftests%2Frequirements.yml&type=code